site/docs/v1/tech/shared/_premium-edition-blurb-api.adoc

```diff
@@ -1 +1 @@
-A paid edition of FusionAuth is required to utilize {premium_feature}.
+icon:radiation-alt[type=fal] [paid]#A paid edition of FusionAuth is required to utilize {premium_feature}.# icon:radiation-alt[type=fal]
```

Comment:
See note from https://github.com/FusionAuth/fusionauth-style/pull/15#issuecomment-856311918 

If we are going to use this to indicate a paid feature, I would rather use our actual SVG for reactor to match the brochure site. We also use this logo in the admin UI. 

![image](https://user-images.githubusercontent.com/1148709/121097105-b16fd580-c7b0-11eb-9b2c-0b0f3ae6e581.png)